roles/deploy_user/tasks/main.yml

```diff
@@ -93,5 +93,5 @@
 # Note that this should not be necessary, but the `restart sshd` handler does not work
 - name: restart sshd for real
   become: true
-  command: systemctl restart sshd
+  ansible.builtin.command: systemctl restart sshd
   when: running_on_server
```

Comment:
Using the `service` module here might fix this. But I think we can replace this with a `flush_handlers` task. Try replacing these four lines with this:
```
  - name: Run all handlers notified by the deploy_user role
    meta: flush_handlers
```
I think we might want this in all our roles, see #2853.

Reply:
I used the flush handlers (because it is the ansible way) but that step is supposed to get skipped by the `running_on_server`